### ISSUE TITLE
parse: Unexpected } (line 1, char 43385)

### ISSUE THREAD

**Issue by mikegleasonjr** [NV/CSSOM#92]
Here's a test case with a `js` and a `css` file:

[test.zip](https://github.com/NV/CSSOM/files/1411149/test.zip)

Basically the `js` is:

```
const cssom = require('cssom');
const fs = require('fs');

fs.readFile('test.css', 'utf8', (err, contents) => {
    console.log(cssom.parse(contents));
});
```

Run with `node test.js`.

**Comment by thomashigginbotham** [comment]
The issue is that your CSS contains an @supports rule. I had the same problem and [created a fix for it](https://github.com/NV/CSSOM/pull/94/commits). I retested on your CSS, and it works fine.

**Comment by NV** [comment]
@mikegleasonjr this should be fixed in cssom v0.3.4. Please verify.
